packages/suite/src/actions/suite/__tests__/metadataActions.test.ts

```diff
@@ -114,8 +114,7 @@ const initStore = (state: State) => {
         // @ts-expect-error
         store.getState().suite = suiteReducer(suite, action);
         store.getState().wallet.accounts = accountsReducer(wallet.accounts, action);
-        store.getState().device = deviceReducer(device, action);
-        // store.
+        store.getState().device = deviceReducer(device, action) as any;
```

Comment:
why is `as any` needed here?

Reply:
The whole device property is typed as `any` and the typing of this file was not strong at all so I just stick to the typing.

It was necessary because the previous reducer is not as strongly typed as `redux-toolkit` reducer is so I just typed it as any not to have to deal with the device property type. I can refactor that if necessary, this was just easy solution not to worry about types in this test file too much.

Reply:
if device is typed as any anyway, it's fine for now in the test